ddtrace/internal/telemetry/writer.py

```diff
@@ -59,65 +101,30 @@ def __init__(self, agent_url=None):
         self._debug = asbool(os.environ.get("DD_TELEMETRY_DEBUG", "false"))
         forksafe.register(self._fork_writer)
 
-        self._headers = {
-            "Content-type": "application/json",
-            "DD-Telemetry-API-Version": "v1",
-            "DD-Telemetry-Debug-Enabled": str(self._debug).lower(),
-        }  # type: Dict[str, str]
-        additional_header_str = os.environ.get("_DD_TELEMETRY_WRITER_ADDITIONAL_HEADERS")
```

Comment:
Even if `_DD_TELEMETRY_WRITER_ADDITIONAL_HEADERS` is set after the `TelemetryWriter` is initialized in tests, do we expect them to not be set by users pre-application startup? Should we be removing this entirely, or is this an internal environment variable?

Reply:
No users will never set this configuration. This configuration is used by the test agent. This code block shouldn't have been added in the first place because it does nothing.